akka-docs/rst/java/code/docs/testkit/ParentChildTest.java

```diff
@@ -45,7 +45,7 @@ public Receive createReceive() {
     public Receive createReceive() {
       return receiveBuilder()
         .matchEquals("ping", message -> {
-          getContext().parent().tell("pong", self());
+          getContext().getParent().tell("pong", self());
```

Comment:
My thinking is that we can't make it "perfect" since we have mixed Scala and Java APIs. We can't hide `context()`, `self()`, `parent()`... since they are inherited from the Scala API. Therefore I was thinking of making the APIs as similar as possible.

BUT, I have no strong opinion about this and we can do it the other way around. Consistency is king, so replace all `self()` -> `getSelf()`, `sender()` -> `getSender()` and `parent()` -> `getParent()`.